docs/repositories-coverage/commits.md

```diff
@@ -76,3 +76,23 @@ The option **Show also files without coverage changes** allows you to list all f
 ![Files tab](images/{{ page.meta.file_name }}-tab-files.png)
 <!-- vale on -->
 <!--tab-files-end-->
+
+<!--tab-diff-start-->
+## Diff tab
+
+The **Diff** tab displays a line-by-line view of the coverage variation introduced by the {{ page.meta.page_name }}. It includes the following areas:
+
+-   A **list of files** modified by the {{ page.meta.page_name }}, with additional information for each file:
+
+    -   A **green plus icon** if the file is added or a **yellow dot icon** if it's modified by the {{ page.meta.page_name }}
+    -   The **coverage variation** introduced by the {{ page.meta.page_name }} or the **total file coverage** if there's no variation
```

Comment:
Correct on both. The variation is green or red.
And I agree that this may indicate a UI that tries to do too much. I'll review this with the squad tomorrow 👍

Reply:
@alerizzo Thoughts on this?

How about simplifying the UI to show only the coverage variation of modified files and a = sign if unchanged, like we do on the Files tab?

Or is there a reason for "compressing" the total coverage value as a default in the UI?

Reply:
@alerizzo After our discussion, I'm OK with keeping this as-is for the first iteration.

@claudiacarpinteiro It's indeed a lot of info, but this is a first iteration. I'm OK with adding the extra information.

Reply:
This is a first iteration and we are testing this in the field, with users. We're gonna need to iterate over this first release, but we want to get our users' feedback on it first. 

We have a lot of information to show, and most of it may help users find what is relevant to look for in a PR when looking to improve coverage. For example, if I add a new file there is no variation; so the total coverage is the important metric to show. If I modified a file, variation is most likely the most important. 

When I was trying to use this page as a developer with my own PRs trying to improve the code coverage metric, both metrics came in handy. But I'm only one developer, so we need to listen now to more users' feedback.